examples/ftd_api_scripts/timerange.py

```diff
@@ -0,0 +1,104 @@
+'''
+Copyright (c) 2019 Cisco and/or its affiliates.
+This software is licensed to you under the terms of the Cisco Sample
+Code License, Version 1.1 (the "License"). A copy of the License
+can be found in the LICENSE.TXT file of this software or at
+https://developer.cisco.com/site/license/cisco-sample-code-license/
+All use of the material herein must be in accordance with the terms of
+the License. All rights not expressly granted by the License are
+reserved. Unless required by applicable law or agreed to separately in
+writing, software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied.
+'''
+
+from ftd_api_resources.timerange_utils import get_all_time_range, post_time_range, update_time_range, get_time_range, delete_time_range
+from ftd_api_resources.access_token import get_access_token
+
+def main():
```

Comment:
Please rename this function and have it accept host, port, user, passwd params, and return true or false depending on outcome. See for example the deploy code.

Reply:
Done